pkgs/applications/misc/coder/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, pkgs, fetchFromGitHub, installShellFiles, makeWrapper, buildGoModule, fetchYarnDeps }:
+
+let
+  pname = "coder";
+  version = "0.10.2";
+
+  src = fetchFromGitHub {
+    owner = "coder";
+    repo = "coder";
+    rev = "v${version}";
+    sha256 = "sha256-h5bN75agNocRAjShbufRCJr45huYJOzCBd4OcGpF4C4=";
+  };
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = src + "/site/yarn.lock";
+    sha256 = "sha256-uDNPRQTpsgxyC5ks+2Qq/wiKjkbjWwSO+cJc5X6qmAA=";
+  };
+in
+buildGoModule rec {
+  inherit pname version src;
+
+  subPackages = [ "cmd/coder" ];
+
+  vendorSha256 = "sha256-+3Zy0zArCXkvD4ogfKdu9W9gJXveAhwFXKG1VRDvOkI=";
+
+  # Flags as provided by the build automation of the project:
+  #   https://github.com/coder/coder/blob/075e891f287b27cdb481a48e129f20a1e6a7de12/scripts/build_go.sh#L89
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/coder/coder/buildinfo.tag=${src.rev}"
```

Comment:
```suggestion
    "-X github.com/coder/coder/buildinfo.tag=${src.version}"
```

Reply:
that does not work

error: attribute 'version' missing

Reply:
I just used the var above